pr_agent/algo/ai_handler.py

```diff
@@ -55,7 +55,7 @@ def deployment_id(self):
 
     @retry(exceptions=(APIError, Timeout, TryAgain, AttributeError, RateLimitError),
            tries=OPENAI_RETRIES, delay=2, backoff=2, jitter=(1, 3))
-    async def chat_completion(self, model: str, temperature: float, system: str, user: str):
+    async def chat_completion(self, model: str, system: str, user: str, temperature: float = 0.2):
         """
         Performs a chat completion using the OpenAI ChatCompletion API.
         Retries in case of API errors or timeouts.
```

Comment:
**Suggestion:** The docstring for the `chat_completion` method should be updated to reflect the new default value for the `temperature` parameter.
```suggestion
    async def chat_completion(self, model: str, system: str, user: str, temperature: float = 0.2):
        """
        Performs a chat completion using the OpenAI ChatCompletion API.
        Retries in case of API errors or timeouts.
        The default temperature is set to 0.2.
```